### PR TITLE
Faculty evaluation 

### PR DESCRIPTION
**Pull request by juliachaloupka**
Added faculty evaluations, faculty navigates to student profile from fstudentlist view and can add evaluation using the "add evaluation" button, after evaluation is submitted, the button is no longer displayed and is replaced with information of their evaluation 

This PR contains commits from @juliachaloupka and @linhpha 

## Issue Related
#214

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Front end change

## Code coverage
97.1%

## Screenshot (if any change in front end)
fviewprofile before evaluation submitted
Screenshot 2019-04-05 13 46 34

fviewprofile after evaluation submitted
Screenshot 2019-04-05 13 47 27

faculty eval view 
Screenshot 2019-04-05 14 04 23

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -95,14 +95,18 @@
 
   create_table "faculty_evaluations", force: :cascade do |t|
     t.integer  "faculty_id"
+    t.integer  "profile_id"
```
Comment (Avanti0532) [inline db/schema.rb]: I would suggest you to pull from master again as your PR doesn't have changes in schema file.

Comment (linhpha) [reply db/schema.rb]: :+1

File: app/views/profiles/fViewProfile.html.erb [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+  <br>
+  <%= link_to "Back", root_path, class: "btn btn-primary"%>
+  <br>
+  <br>
+  <%if !FacultyEvaluation.exists?(application_id: @application.id ) %>
+    <i>Evaluation for this application has not been submitted</i><br>
+    <%= link_to "Add Evaluation", faculty_evaluation_path(@application.id), class: "btn btn-large btn-success"%>
+  <%else %>
+    <% evaluation = FacultyEvaluation.where(:application_id => @application.id).first %>
+    <i style="color: #2fb471" class="glyphicon glyphicon-ok"></i>
```
Comment (hpitawela) [inline app/views/profiles/fViewProfile.html.erb]: Better move these styles to the relevant css file.

Comment (linhpha) [reply app/views/profiles/fViewProfile.html.erb]: :+1:

File: db/schema.rb [hunk 3]
```diff
@@ -95,7 +95,6 @@
 
   create_table "faculty_evaluations", force: :cascade do |t|
     t.integer  "faculty_id"
-    t.integer  "profile_id"
```
Comment (Avanti0532) [inline db/schema.rb]: Can you add separate migration for removing profile_id from faculty evaluation table?

**Pull request merged**